include/splitvector/splitvec.h

```diff
@@ -1032,10 +1051,11 @@ class SplitVector {
    void push_back(const T& val) {
       // If we have no allocated memory because the default ctor was used then
       // allocate one element, set it and return
-      if (_data == nullptr) {
-         *this = SplitVector<T, Allocator>(1, val);
-         return;
-      }
+      // if (_data == nullptr) {
```

Comment:
why won't this be an issue anymore? (same question on line 1074)

Reply:
This is because the default constructor changed.